python/cudf/cudf/core/frame.py

```diff
@@ -2693,7 +2693,15 @@ def acos(self):
                     1.5707963267948966,  1.266103672779499],
                     dtype='float64')
         """
-        return self._unaryop("acos")
+        result = self.copy()
```

Comment:
This is a deep copy of the data which is expensive, could we use a shallow copy here instead?

Reply:
Yes, we can use shallow copy here instead, @marlenezw could you switch it to shallow copy? (pass `deep=False`)

Reply:
Sounds good, I will!